plugin/adsk/plugin/adskImportCommand.h

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef ADSK_MAYA_IMPORT_COMMAND_H
+#define ADSK_MAYA_IMPORT_COMMAND_H
+
+#include "base/api.h"

```

Comment:
include as `<mayaUsd/base/api.h>`

Reply:
I believe it's a different api.h header, see https://github.com/Autodesk/maya-usd/blob/dev/plugin/adsk/plugin/base/api.h 
So it should be `<mayaUsdPlugin/base/api.h>`

Reply:
We no longer promote private headers and therefore  `#include "base/api.h" `  is correct code. Please see latest changes in 3324c613c

Reply:
Ha, good catch @HamedSabri-adsk, I wasn't aware it's a private header in the plugin. Then all is good.

Reply:
Ahh, my bad. Sorry about that! Thanks @kxl-adsk and @HamedSabri-adsk!